nvflare/app_opt/flower/flower_job.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os.path
+from typing import List, Optional
+
+from nvflare.app_common.tie.defs import Constant
+from nvflare.job_config.api import ControllerApp, ExecutorApp, FedJob
+
+from .controller import FlowerController
+from .executor import FlowerExecutor
+
+
+class FlowerJob(FedJob):
+    def __init__(
+        self,
+        job_name: str,
+        flower_content: str,
+        min_clients: int = 1,
+        mandatory_clients: Optional[List[str]] = None,
+        database: str = "",
+        server_app_args: list = None,
+        superlink_ready_timeout: float = 10.0,
+        configure_task_timeout=Constant.CONFIG_TASK_TIMEOUT,
+        start_task_timeout=Constant.START_TASK_TIMEOUT,
+        max_client_op_interval: float = Constant.MAX_CLIENT_OP_INTERVAL,
+        progress_timeout: float = Constant.WORKFLOW_PROGRESS_TIMEOUT,
+        per_msg_timeout=10.0,
+        tx_timeout=100.0,
+        client_shutdown_timeout=5.0,
+    ):
```

Comment:
since this object is user facing, could we add docstring to it and explain the arguments?